powershellgroup/powershellgroup.resource.ps1

```diff
@@ -299,22 +299,26 @@ elseif ($Operation -eq 'Test')
         if ($cachedResourceInfo)
         {
             $inputht = @{}
-            $ResourceTypeName = ($inputobj_pscustomobj.type -split "/")[1]
-            $inputobj_pscustomobj.psobject.properties | %{ 
-                if ($_.Name -ne "type")
-                {
-                    $inputht[$_.Name] = $_.Value
+            $ResourceTypeName = ($r.type -split "/")[1]
+            $r.properties.psobject.properties | %{ $inputht[$_.Name] = $_.Value }
```

Comment:
The input object has field `type` injected into it by `dsc`. This field has to be filtered out before passing the object down to `Invoke-DscResource` calls.